src/main/java/module/lineup/LineupTableModel.java

```diff
@@ -1,139 +1,129 @@
 package module.lineup;
 
-import core.gui.comp.table.BooleanColumn;
-import core.gui.comp.table.HOTableModel;
-import core.gui.comp.table.UserColumn;
-import core.gui.model.PlayerColumn;
+import core.gui.model.PlayerOverviewTableModel;
 import core.gui.model.UserColumnController;
-import core.gui.model.UserColumnFactory;
-import core.model.player.Player;
-import org.jetbrains.annotations.Nullable;
 
-import java.util.ArrayList;
-import java.util.List;
 
+public class LineupTableModel extends PlayerOverviewTableModel {
```

Comment:
Is there any point to this class at all now?  or is it needed to set the id, and therefore save the state?

Reply:
Removed this class